query.go

```diff
@@ -552,11 +564,22 @@ func (qr *QueryResult) Scan(dest ...interface{}) error {
 					return err
 				}
 				*d = b
+			case bool:
+				*d = src
 			case nil:
 				trace("%s: skipping nil scan data for variable #%d (%s)", qr.conn.ID, n, qr.columns[n])
 			default:
 				return fmt.Errorf("invalid bool col:%d type:%T val:%v", n, src, src)
 			}
+		case *[]uint8:
+			switch src := src.(type) {
+			case []uint8:
```

Comment:
I don't believe `src` can ever be `uint8`. As far as I know, it will either be a `int64`, `float64`, `string`, or `nil`. It might be good to handle all of those cases similar to other types.